### PR TITLE
es2015-fy tests, readme + xo

### PR DESCRIPTION
**Pull request by dsblv**
Bumped `AVA`, es2015-fied test & readme, added `xo`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,7 +42,8 @@
     "convert"
   ],
   "devDependencies": {
-    "ava": "0.0.4"
+    "ava": "^0.3.0",
```
Comment (sindresorhus) [inline package.json]: Can you set `ava` to `"*"` too?

File: test.js [hunk 2]
```diff
@@ -1,8 +1,7 @@
-'use strict';
-var test = require('ava');
-var camelcaseKeys = require('./');
+import test from 'ava';
+import camelcaseKeys from './';
```
Comment (sindresorhus) [inline test.js]: `import fn from './';`

File: test.js [hunk 3]
```diff
@@ -1,8 +1,7 @@
...
 
-test(function (t) {
-	t.assert(camelcaseKeys({'foo-bar': true}).fooBar);
+test('camelcase keys', t => {
```
Comment (sindresorhus) [inline test.js]: I prefer to not use a title when there's only one test.

File: test.js [hunk 4]
```diff
@@ -1,8 +1,7 @@
...
+	t.ok(camelcaseKeys({'foo-bar': true}).fooBar);
```
Comment (sindresorhus) [inline test.js]: `t.ok` => `t.true`

File: readme.md [hunk 5]
```diff
@@ -13,13 +13,13 @@ $ npm install --save camelcase-keys
 ## Usage
```
Comment (sindresorhus) [inline readme.md]: Can you also remove the above `sh`?

Comment (dsblv) [inline readme.md]: Done :ok_hand:

**Comment by sindresorhus** [comment]
Perfect. Thank you :)

File: test.js (hunk 3 above)
Comment (dsblv) [inline test.js]: `✓ [anonimous]` is kind of strange, would be cool if `AVA` knew what's up and called the only test `default` for example. What do you think?

Comment (sindresorhus) [inline test.js]: Agreed. It used to be like that, but was regressed: https://github.com/sindresorhus/ava/issues/112

**Pull request closed without merging**